runtime/onert/api/nnfw/src/nnfw_api_internal.h

```diff
@@ -182,6 +182,8 @@ struct nnfw_session
   NNFW_STATUS train_set_expected(uint32_t index, const void *expected,
                                  const nnfw_tensorinfo *expected_tensorinfo);
   NNFW_STATUS train_set_output(uint32_t index, NNFW_TYPE type, void *buffer, size_t length);
+  NNFW_STATUS nnfw_train_disable_node_update(uint32_t op_index);
+  NNFW_STATUS nnfw_train_enable_node_update(uint32_t op_index);
```

Comment:
```suggestion
  NNFW_STATUS train_disable_node_update(uint32_t op_index);
  NNFW_STATUS train_enable_node_update(uint32_t op_index);
```

Reply:
not a problem after applying https://github.com/Samsung/ONE/pull/13094#discussion_r1625154291 ;)